cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/storage/SkuTypeResolver.java

```diff
@@ -0,0 +1,27 @@
+package com.sequenceiq.cloudbreak.cloud.azure.storage;
+
+import org.springframework.stereotype.Component;
+
+import com.microsoft.azure.management.storage.SkuName;
+import com.microsoft.azure.management.storage.StorageAccountSkuType;
+import com.sequenceiq.cloudbreak.cloud.azure.AzureDiskType;
+
+@Component
+public class SkuTypeResolver {
+
+    public StorageAccountSkuType resolveFromAzureDiskType(AzureDiskType type) {
+        StorageAccountSkuType result = null;
```

Comment:
```
return Optional.ofNullable(type)
.map(type -> type.value().split("_"))
.map(typeComponents -> {
String prefix = "";
if (typeComponents[0].toUpperCase().contains("STANDARD")) {
                 prefix = "STANDARD";
             } else if (typeComponents[0].toUpperCase().contains("PREMIUM")) {
                 prefix = "PREMIUM";
             }
return prefix + "_" + typeComponents[typeComponents.length - 1]
})
.map(x -> StorageAccountSkuType.fromSkuName(SkuName.fromString(x)))
.get()
```

Reply:
you're right, but does it make any performance improvement to change it that way? If not, I wouldn't wrap a clearly readable code to a bunch of lambdas.